tests/test-suite/test-data/snake_case/seeds/seed.ts

```diff
@@ -1,3 +1,5 @@
+let faker = require('faker');
```

Comment:
@isBatak This should probably be `const` right? Unless I'm missing a reassignment somewhere?

Reply:
It seems like `requires` are shared between `snake_case` and `camelCase` seeds, so this was a workaround.
You can see in `camelCase` seeds I'm requiring it like this `faker = require('faker');`, basically reassigning the variable, that's why it's `let`. 
It's not ideal but it works. Also, it would be cool if we can use `import faker from 'faker';` syntax but it is not working inside seed files for some reason.

Reply:
@spersico Any ideas here? I'm guessing it has something to do with the way we load the seed files?